### ISSUE TITLE
Add height base media query function

### ISSUE THREAD

**Issue by perruche** [studiometa/scss-toolkit#4]
Hey, 

We often have to use custom based media query heights for each project. 
Currently we write it like regular css `@media screen and (min-height: 48em) {}`

Should we create a scss function which work like `md` ? 
Maybe something like `@media #{mdh('s')} {}` ?

**Comment by titouanmathis** [comment]
Could a new parameter to the `media($breakoint, $type, $unit)` function be enough?

We could have something looking like this:
```scss
@function media($breakpoint, $type: 'min', $unit: 'em', $orientation: 'width') {
  // ...
  @return '(min-#{$orientation}: #{$size})';
}
```

**Comment by lusimeon** [comment]
@titouanmathis it seems to be a good idea.

Furthermor, I think we should have a specific height based breakpoint map related to this function.

``` scss
$breakpoints-height: (
  'xxs': ..,
  'xs': ..,
  's': ..,
  'm': ..
  'l': ..
  'xl': ..,
  'xxl': ..,
) !default;
```

**Comment by titouanmathis** [comment]
Fixed by #17.
